docs/appendices/release-notes/5.5.2.rst

```diff
@@ -46,6 +46,39 @@ See the :ref:`version_5.5.0` release notes for a full list of changes in the
 5.5 series.
 
 
+Packaging Changes
+=================
+
+- The RPM and DEB packages changed slightly to unify the build process. The most
+  important change is that the ``crate`` service no longer automatically starts
+  after package installation, to allow changing the configuration first.
+
+  Other than that, the structure is now:
+
+    - ``bin``, ``jdk`` and ``lib`` are installed into ``/usr/share/crate``. In the RPM
+      package this used to be in ``/opt/crate``.
+
+    - The home directory of the crate user is ``/usr/share/crate``
+
+    - changes, notice, license are in ``/usr/share/doc/crate``
+
+    - service file is in ``/usr/lib/systemd/system``
+
+    - The ``crate.yml`` configuration file is in ``/etc/crate/``
```

Comment:
I guess this didn't change even for RPM (because you added an explicit comment above for RPM)?

Reply:
Yes it didn't change, but thought I'd add it to give a complete overview. Should I remove it?

Reply:
👍 I like the overview